web/packages/teleport/src/Integrations/status/AwsOidc/StatCard.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+
+import { Card, Flex, H2, Text } from 'design';
+import { ResourceIcon } from 'design/ResourceIcon';
+
+import * as Icons from 'design/Icon';
+
+import { formatDistanceStrict } from 'date-fns';
+
+import { ResourceTypeSummary } from 'teleport/services/integrations';
+
+export enum AwsResource {
+  ec2 = 'ec2',
+  eks = 'eks',
+  rds = 'rds',
+}
+
+type StatCardProps = {
+  resource: AwsResource;
+  summary?: ResourceTypeSummary;
+};
+
+export function StatCard({ resource, summary }: StatCardProps) {
+  const updated = summary?.discoverLastSync
+    ? new Date(summary?.discoverLastSync)
+    : undefined;
+
+  const term =
+    resource == AwsResource.rds
+      ? 'Databases'
+      : resource == AwsResource.eks
+        ? 'Clusters'
+        : 'Instances';
```

Comment:
Thoughts on adding `resourceToTerm()` that has a `switch` on `AwsResource`? Bit nicer than nested ternaries and will be easier to expand in the future